Svc/FileManager/test/int/test_cmd_fileManager.py

```diff
@@ -0,0 +1,44 @@
+""" test_cmd_FileManager.py:
+
+Test the command FileManager with basic integration tests.
+    fileManager.CreateDirectory
+    fileManager.ShellCommand
+    fileManager.FileSize
+    fileManager.MoveFile
+    fileManager.AppendFile
+    fileManager.RemoveFile
+    fileManager.RemoveDirectory
+
+"""
+
+
+def test_send_command(fprime_test_api):
+    """Test that commands may be sent
+
+    Tests command send, dispatch, and receipt using send_and_assert command with a pair of fileManager commands.
+    """
+    fprime_test_api.send_and_assert_command(fprime_test_api.get_mnemonic('Svc.FileManager') + '.' + 'CreateDirectory', ["/tmp/file"], max_delay=1)
+
+    fprime_test_api.send_and_assert_command(fprime_test_api.get_mnemonic('Svc.FileManager') + '.' + 'CreateDirectory', ["/tmp/file2"], max_delay=1)
+
+    fprime_test_api.send_and_assert_command(fprime_test_api.get_mnemonic('Svc.FileManager') + '.' + 'ShellCommand', ["ls", "/tmp/shell_ls.log"], max_delay=1)
```

Comment:
Don't use ShellCommand. If you want to create a file, you can create a dummy file in this test script and upload it to the spacecraft. Then you can test with that uploaded file.